src/main/java/racinggame/domain/Car.java

```diff
@@ -0,0 +1,71 @@
+package racinggame.domain;
+
+import static racinggame.view.OutputView.*;
+
+public class Car {
+
+    public static final int MIN_MOVE_NUMBER = 0;
+    public static final int MAX_MOVE_NUMBER = 9;
+    public static final int MOVE_THRESHOLD = 4;
+    public static final int MAX_NAME_LENGTH = 5;
+
+    private int position;
+    private final String name;
+
+    public Car() {
+        this.name = "";
+        this.position = 0;
```

Comment:
```suggestion
        this("");
```
부 생성자가 주 생성자를 호출하는 방식으로 구현하는 습관 들일 것을 추천
그런데 자동차 이름이 없으면 아예 Car가 생성되면 안될 것 같은데 이 생성자가 필요할까?